sql/analyzer/aggregations.go

```diff
@@ -75,13 +75,18 @@ func flattenedGroupBy(ctx *sql.Context, projection, grouping []sql.Expression, c
 func replaceAggregatesWithGetFieldProjections(ctx *sql.Context, projection []sql.Expression) (projections, aggregations []sql.Expression, err error) {
 	var newProjection = make([]sql.Expression, len(projection))
 	var newAggregates []sql.Expression
-
+	allGetFields := make(map[int]sql.Expression, 0)
```

Comment:
No need to declare size zero in map initializers like this.

Generally speaking the only time you want to explicitly size a map or slice is when you know ahead of time how big it will be.